pkg/apis/servicecatalog/checksum/unversioned/checksum.go

```diff
@@ -37,7 +37,12 @@ func InstanceSpecChecksum(spec servicecatalog.InstanceSpec) string {
 	specString += fmt.Sprintf("planName: %v\n", spec.PlanName)
 
 	if spec.Parameters != nil {
-		specString += fmt.Sprintf("parameters:\n\n%v\n\n", string(spec.Parameters.Raw))
+		params := spec.Parameters
+		if params.SecretRef != nil {
+			specString += fmt.Sprintf("parameters.secretRef: %v\n", params.SecretRef.Name)
```

Comment:
@duglin that's right, see the comment: https://github.com/kubernetes-incubator/service-catalog/pull/1028#discussion_r127391333
I can add a comment with a link to a separate issue there.